s4e-web/src/app/views/map-view/view-manager/overlay-list-modal/wms-url.utils.ts

```diff
@@ -0,0 +1,237 @@
+import { AbstractControl } from '@angular/forms';
+
+const URL_EXPRESSION = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/i;
```

Comment:
change URL to accept
- IP with port
- localhost page with port
- normal page with port